models/sushiswap/arbitrum/sushiswap_arbitrum_trades.sql

```diff
@@ -1,10 +1,11 @@
 {{ config(
-    alias = alias('trades')
+    tags=['dunesql']
+    ,alias = alias('trades')
     ,partition_by = ['block_date']
```

Comment:
same idea on partition here as from pancake swap (and applies universally in this PR)